extensions/amp-video-iframe/1.0/component.js

```diff
@@ -1 +1,2 @@
-export {VideoIframe} from '../../amp-video/1.0/video-iframe';
+// Expose from package as BentoVideoIframe
+export {VideoIframe as BentoVideoIframe} from '../../amp-video/1.0/video-iframe';
```

Comment:
Why not rename it in the function declaration?
Also, we will need to update names in this doc: https://github.com/ampproject/amphtml/blob/main/docs/building-a-bento-video-player.md

Reply:
The question is, at what level do we include `Bento`?

It seems to be that if we rename `VideoIframe` we should also rename `VideoWrapper` for consistency. Then, what about `ProxyFrame`, etc?

Reply:
Discussed offline - 
`Bento` prefix is for released components
Non-prefixed components are used for composition internally